.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_DESCRIPTION.md

```diff
@@ -0,0 +1,52 @@
+# GreenCityUBS PR
```

Comment:
You do not need to change of modify PULL_REQUEST template
Toy just need to fill it in when creating the PR